### ISSUE TITLE
Heading level support

### ISSUE THREAD

**Issue by Fred-Vatin** [spywhere/vscode-mark-jump#2]
Hi,

We have talked about similar feature in this [VS Code](https://github.com/Microsoft/vscode/issues/23698) issue and your extension came up.

Could you add heading level support? like that:
![Image](https://camo.githubusercontent.com/dbb8910ec51af12b0874103c8b61a07628962930/687474703a2f2f696d6775722e636f6d2f754968735138412e676966)

**Comment by spywhere** [comment]
First of all, really sorry for such a long delay to response. As I didn't watched this repository just because it's seem to me like I'm liking my own project and such. So, yeah, enough excuse.

What you mean is to support another type of bookmark in addition to ToDo, and Note, right? Or just another section mark with different syntax? If it's the latter, Mark Jump already support it as you can configure the pattern used for mark the section. Just a look at the extension's configurations it self. Keep in mind that the patterns used are case-sensitive, so if you want to match a case-insensitive syntax you have to make sure your patterns support it.

**Comment by Fred-Vatin** [comment]
@spywhere Sorry too for long delay response. I've been kept away from coding for a moment.

As you can see in the picture in the first post, the goal is to implement mark levels that you can see in the widget (command panel) as:

```
Head 1
- Head 2
-- Head 3
```

I think supporting 3 levels is enough in comments. The idea is to make possible to render level tree in panel.

For instance to define a mark level in comment:

```
// > Level 1
// >> Level 2
// >>> Level 3
```

 See this sublime [addon](https://github.com/kizza/Table-of-comments) for more details. It produces a TOC in top of document summarizing all the comment marks. It will be even greater to support that.

**Comment by spywhere** [comment]
Got it. I'll keep you dated once I got it working.

**Comment by spywhere** [comment]
This feature has been implemented in v0.4.0.

![screen shot 2560-06-14 at 00 09 02](https://user-images.githubusercontent.com/1087399/27094835-bd796ea4-5095-11e7-98d2-db0728b0b7d2.png)
